modules/http/src/main/java/org/mule/module/http/internal/config/HttpNamespaceHandler.java

```diff
@@ -66,6 +67,8 @@ public void init()
 
         registerBeanDefinitionParser("raml-api-configuration", new ChildDefinitionParser("apiConfiguration", RamlApiConfiguration.class));
         registerBeanDefinitionParser("worker-threading-profile", new ThreadingProfileDefinitionParser("workerThreadingProfile",  MuleProperties.OBJECT_DEFAULT_MESSAGE_RECEIVER_THREADING_PROFILE));
+        registerBeanDefinitionParser("direct-threading-profile", new ThreadingProfileDefinitionParser("workerThreadingProfile",  MuleProperties.OBJECT_DEFAULT_MESSAGE_RECEIVER_THREADING_PROFILE,
```

Comment:
I hadn't change this to not change method on HttpListenerConfig.  But i've just look at it's not in interface, so will update.